content/en/docs/concepts/storage/volumes.md

```diff
@@ -720,13 +720,12 @@ For more details, see the [Portworx volume](https://github.com/kubernetes/exampl
 #### Portworx CSI migration
 {{< feature-state for_k8s_version="v1.25" state="beta" >}}
 
-The `CSIMigration` feature for Portworx has been added but disabled by default in Kubernetes 1.23 since it's in alpha state.
-It has been beta now since v1.25 but it is still turned off by default.
+The `CSIMigration` feature for Portworx has been added but disabled by default in Kubernetes 1.23 since it's in alpha state. It was promoted to beta in Kubernetes 1.25 but it was turned off by default. Since Kubernetes 1.31, the feature is turned on by default. If you want to disable the feature, you can set the `CSIMigrationPortworx` feature flag to `false` in the kube-controller-manager and kubelet.
```

Comment:
```suggestion
By default, Kubernetes {{% skew currentVersion %}} attempts to migrate legacy
Portworx volumes to use CSI. (CSI migration for Portworks has been available since
Kubernetes v1.23, but was only turned on by default with the v1.31 release).
If you want to disable automatic migration, you can set the `CSIMigrationPortworx`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
to `false`; you need to make that change for the kube-controller-manager **and** on
every relevant kubelet.
```